policyengine_us/variables/gov/states/md/tax/income/capital_gains/md_capital_gains_surtax.py

```diff
@@ -0,0 +1,79 @@
+from policyengine_us.model_api import *
+
+
+class md_capital_gains_surtax(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Maryland capital gains surtax"
+    definition_period = YEAR
+    unit = USD
+    reference = [
+        "https://mgaleg.maryland.gov/2025RS/Chapters_noln/CH_604_hb0352e.pdf#page=162"  # Maryland House Bill 352 - Budget Reconciliation and Financing Act of 2025
+    ]
+    defined_for = "md_capital_gains_surtax_applies"
+
+    def formula(tax_unit, period, parameters):
+
+        p = parameters(period).gov.states.md.tax.income.capital_gains
+
+        # Get net capital gains (sum of long-term and short-term)
+        # NOTE: This is the basic implementation using readily available variables
+        long_term_gains = add(tax_unit, period, ["long_term_capital_gains"])
+        short_term_gains = add(tax_unit, period, ["short_term_capital_gains"])
+        total_capital_gains = long_term_gains + short_term_gains
+
+        # Apply capital losses (with federal $3,000 limitation)
+        capital_losses = add(tax_unit, period, ["capital_losses"])
```

Comment:
I dont think `capital_losses` are capped in our model? Did we miss that? 

Also please avoid using numeric values in the comments. It can get confusing when the law changes but the comment stays the same

```suggestion
        # Apply capital losses (with federal limitation)
        capital_losses = add(tax_unit, period, ["capital_losses"])
```